src/main/antlr4/Manifold.g4

```diff
@@ -55,7 +55,7 @@ expression:
   | namespacedIdentifier # VariableReferenceExpression
   | expression '=' expression # AssignmentExpression
   | 'primitive' 'port' typevalue (':' tupleTypeValue)? # PrimitivePortDefinitionExpression
-  | 'primitive' 'node' functionTypeValue (':' tupleTypeValue)? # PrimitiveNodeDefinitionExpression
+  | 'primitive' 'node' functionTypeValue # PrimitiveNodeDefinitionExpression
```

Comment:
I won't get too nitpicky about this because I'm still of the opinion that we should define primitives in an intermediate-like JSON file provided by the backend.